keps/sig-node/2727-grpc-probe/kep.yaml

```diff
@@ -0,0 +1,24 @@
+title: Add gRPC probe to Pod.Spec.Container.{Liveness,Readiness,Startup}Probe
+kep-number: 2727
+authors:
+  - "@bowei"
+  - "@PxyUp"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-network
+status: implementable
+creation-date: 2020-04-04
+modified-date: 2021-05-12
+reviewers:
+  - "@thockin"
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@thockin"
+see-also:
+replaces:
+prr-approvers:
+  - "@johnbelarmic"
+stage: "alpha"
+latest-milestone: "v1.22"
```

Comment:
There's some metadata missing here (for example feature gate info), PTAL at the latest kep.yaml template (where you got the PRR section from). Won't block on this but I would like to see it here ASAP.